mlflow/pyfunc/model.py

```diff
@@ -373,27 +374,11 @@ def _save_model_with_class_artifacts_params(  # noqa: D417
             with open(os.path.join(path, saved_python_model_subpath), "wb") as out:
                 cloudpickle.dump(python_model, out)
         except Exception as e:
-            # cloudpickle sometimes raises TypeError instead of PicklingError.
-            # catching generic Exception and checking message to handle both cases.
-            if "cannot pickle" in str(e).lower():
-                raise MlflowException(
-                    "Failed to serialize Python model. Please audit your "
-                    "class variables (e.g. in `__init__()`) for any "
-                    "unpicklable objects. If you're trying to save an external model "
-                    "in your custom pyfunc, Please use the `artifacts` parameter "
-                    "in `mlflow.pyfunc.save_model()`, and load your external model "
-                    "in the `load_context()` method instead. For example:\n\n"
-                    "class MyModel(mlflow.pyfunc.PythonModel):\n"
-                    "    def load_context(self, context):\n"
-                    "        model_path = context.artifacts['my_model_path']\n"
-                    "        // custom load logic here\n"
-                    "        self.model = load_model(model_path)\n\n"
-                    "For more information, see our full tutorial at: "
-                    "https://mlflow.org/docs/latest/traditional-ml/creating-custom-pyfunc/index.html"
-                    f"\n\nFull serialization error: {e}"
-                ) from None
-            else:
-                raise e
+            raise MlflowException(
+                "Failed to serialize Python model. Please save the model as a python file "
```

Comment:
```suggestion
                "Failed to serialize Python model. Please log the model as code"
```

can we just say `log the model as code`?

Reply:
I think `save the model into a python file` makes more sense? I'm not sure what `log the model as code` really means 🤔